python/torch_mlir_e2e_test/test_suite/rng.py

```diff
@@ -364,3 +364,46 @@ def forward(self):
 @register_test_case(module_factory=lambda: RandnGeneratorModule())
 def RandnGeneratorModule_basic(module, tu: TestUtils):
     module.forward()
+
+
+# ==============================================================================
+
+class RandnLikeModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float64, True),
+    ])
+    def forward(self, x):
+        a = torch.ops.aten.randn_like(x)
+        std = torch.std(a)
+        return std
+
+
+@register_test_case(module_factory=lambda: RandnLikeModule())
+def RandnLikeModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(4, 512, 1024).double())
+
+# ==============================================================================
+
+class RandnLikeDtypeModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float64, True),
+    ])
+    def forward(self, x):
+        a = torch.ops.aten.randn_like(x, dtype=torch.float32)
```

Comment:
Any reason we can pass x as type `torch.float32`?

Reply:
`torch.float64` is passed as the default value for numerical stability of the result of the mean/std op for large-sized tensors.